src/Controller/OcrController.php

```diff
@@ -177,8 +178,40 @@ public function homeAction(): Response
     }
 
     /**
-     * @Route("/api", name="api")
-     * @Route("/api.php", name="apiPhp")
+     * @Route("/api", name="api", methods={"GET", "POST"})
+     * @Route("/api.php", name="apiPhp", methods={"GET", "POST"})
+     * @OA\Parameter(
+     *     name="engine",
+     *     in="query",
+     *     description="The engine to use, either `tesseract` or `google`.",
+     *     @OA\Schema(type="string")
+     * )
+     * @OA\Parameter(
+     *     name="image",
+     *     in="query",
+     *     description="The image URL.",
+     *     @OA\Schema(type="string")
+     * )
+     * @OA\Parameter(
+     *     name="langs",
+     *     in="query",
+     *     description="List of language codes.
+     *         Can be left empty, in which case the engine will do its best (useful for unsupported languages).",
```

Comment:
This asterisk is appearing verbatim in the documentation. I think it doesn't support line breaks.

Reply:
Found https://github.com/zircote/swagger-php/issues/326